spk/rclone/Makefile

```
@@ -0,0 +1,25 @@
+SPK_NAME = rclone
+SPK_VERS = 1.69.1
+SPK_REV = 1
+SPK_ICON = src/logo.png
+
+DEPENDS = cross/$(SPK_NAME)
```

since #6255 we must not use variables in DEPENDS anymore
```suggestion
DEPENDS = cross/rclone
```